Jenkinsfile_CNP

```diff
@@ -33,6 +32,13 @@ def secrets = [
         secret('ccd-private-cross-case-type-solicitor-password', 'CCD_PRIVATE_CROSS_CASE_TYPE_SOLICITOR_PASSWORD'),
         secret('ccd-restricted-cross-case-type-caseworker-email', 'CCD_RESTRICTED_CROSS_CASE_TYPE_CASEWORKER_EMAIL'),
         secret('ccd-restricted-cross-case-type-caseworker-password', 'CCD_RESTRICTED_CROSS_CASE_TYPE_CASEWORKER_PASSWORD'),
+        secret('ccd-cache-warm-up-email', 'CCD_CACHE_WARM_UP_EMAIL'),
+        secret('ccd-cache-warm-up-password', 'CCD_CACHE_WARM_UP_PASSWORD'),
+        secret('ccd-cache-warm-up-sleep-time-sec', 'CCD_CACHE_WARM_UP_SLEEP_TIME_SEC'),
```

Comment:
rather than having `ccd-cache-warm-up-sleep-time-sec` and `ccd-cache-warm-up-enabled` declared as secrets,
can't we have them like terraform variables in the `main.tf` and overriding them per env? Like we do for example for `DEFINITION_CACHE_TTL_SEC`

Reply:
Done